snuba/web/views.py

```diff
@@ -37,14 +37,13 @@
 from snuba.datasets.dataset import Dataset
 from snuba.datasets.factory import (
     InvalidDatasetError,
-    enforce_table_writer,
     get_dataset,
     get_dataset_name,
     get_enabled_dataset_names,
 )
 from snuba.datasets.schemas.tables import TableSchema
 from snuba.query.composite import CompositeQuery
-from snuba.query.data_source.simple import Entity
+from snuba.query.data_source.simple import Entity as EntityMarker
```

Comment:
This is a nit, but I would rather leave these imports the same and have the aliases in the testing code.